CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.22.1](https://github.com/GetStream/stream-chat-ruby/compare/v2.22.0...v2.22.1) (2022-04-20)
+
```

Comment:
I should have added a `feat` not a `chore` 🤦 Chore's don't show up in the changelog. I'll add it manually

Reply:
✅